test/compilable/b20067.d

```diff
@@ -0,0 +1,21 @@
+struct S1
+{
+    int i;
+    @property int ii(){return 0;}
+    @property bool empty(){return true;}
+    alias front = ii;
+    void popFront(){}
+}
+struct S2
+{
+    int i;
+    @property bool empty(){return true;}
```

Comment:
Can you add/modify a test to include an alias also for `empty` for completeness?

Reply:
done

Reply:
Thanks!